debian/debathena-pyhesiodfs.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Hesiod automounter for Athena lockers
+Requires=openafs-client.service
+After=local-fs.target network.target openafs-client.service
+
+[Service]
+Type=simple
+Environment="pyhesiodfs_dir=/mit"
+OOMScoreAdjust=-1000
+ExecStartPre=/bin/mkdir -p /mit
```

Comment:
What I’m pointing out is an inconsistency between the `mkdir` line, which uses `/mit` literally, and all the other lines that use the variable.